bin/interpreter.sh

```diff
@@ -204,6 +204,21 @@ elif [[ "${INTERPRETER_ID}" == "pig" ]]; then
   else
     echo "TEZ_CONF_DIR is not set, configuration might not be loaded"
   fi
+elif [[ "${INTERPRETER_ID}" == "flink" ]]; then
+  if [[ -n "${HADOOP_CONF_DIR}" ]] && [[ -d "${HADOOP_CONF_DIR}" ]]; then
+    ZEPPELIN_INTP_CLASSPATH+=":${HADOOP_CONF_DIR}"
+    export HADOOP_CONF_DIR=${HADOOP_CONF_DIR}
```

Comment:
why do we need this if it's already set?

Reply:
flink yarn mode needs `HADOOP_CONF_DIR`

Reply:
`if [[ -n "${HADOOP_CONF_DIR}" ]]` this says `HADOOP_CONF_DIR` is already in the environment?

Reply:
yes

Reply:
so then why export the exact same thing again?